content/en/database_monitoring/guide/tag_database_statements.md

```diff
@@ -18,12 +18,9 @@ Supported databases
 Supported Agent versions
 : 7.36.1+
 
-Supported tagging formats
-: [sqlcommenter][2], [marginalia][3]
-
 
 ## Manual tag injection
-Using any database API supporting execution of SQL statements, add a comment in your statement with tags formatted as per the [sqlcommenter][2] or [marginalia][3] formats.
+Using any database API supporting execution of SQL statements, add a comment in your statement with tags formatted as `<YOUR_KEY>:'<YOUR_VALUE>'`. The value must be in quotes.
```

Comment:
I don't think this change is correct. Database monitoring parses sqlcommenter and marginalia formats and that was correctly stated in the documentation. If there's a problem parsing either spec, we should address and fix it on our side.